kview/src/main/java/dev/ikm/komet/kview/mvvm/view/journal/JournalController.java

```diff
@@ -358,6 +375,12 @@ public void initialize() {
         };
         journalEventBus.subscribe(JOURNAL_TOPIC, MakeConceptWindowEvent.class, makeConceptWindowEventSubscriber);
 
+        makePatternWindowEventSubscriber = evt -> {
+            LOG.info("FIXME... edit pattern window");
+            makePatternWindow(evt.getPatternFacade(), evt.getViewProperties());
```

Comment:
(Optional) not a nit pick, but since the `makePatternWindow()` method is inside of the journal controller, it has access to a **view properties**. Does it really need it?

Reply:
I have to look but I think it does...